### PR TITLE
Don't list datasets without environment

### PR DESCRIPTION
**Pull request by Yrob**

### CODE REVIEW

File: static/dso_tables.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
  */
 
  const domain = window.location.origin;
- const dsoPath = "/v1/";
+ const dsoPath = "localhost:8000/v1/";
```
Comment (jjmurre) [inline static/dso_tables.js:6]: Gaat `localhost` werken op ACC en PRD?

File: static/dso_tables.js [hunk 2]
```diff
@@ -124,10 +124,10 @@
  
  window.onload = () => {
     let promises = [
-        JSONRequest("/api/manual_apis.json").catch(e => {console.log("Kan manual datasets niet ophalen.")}),
         JSONRequest(domain + dsoPath + "?_format=json").catch(e => {console.log("Kan datasets niet ophalen.")}),
-        JSONRequest(domain + dsoPath + "wfs/").catch(e => {console.log("Kan mvt datasets niet ophalen.")}),
-        JSONRequest(domain + dsoPath + "mvt/").catch(e => {console.log("Kan wfs datasets niet ophalen.")})
+    JSONRequest(domain + dsoPath + "wfs/?_format=json").catch(e => {console.log("Kan mvt datasets niet ophalen.")}),
+    JSONRequest(domain + dsoPath + "mvt/?_format=json").catch(e => {console.log("Kan wfs datasets niet ophalen.")}),
+        JSONRequest("/api/manual_apis.json").catch(e => {console.log("Kan manual datasets niet ophalen.")}),
```
Comment (lbam) [inline static/dso_tables.js:130]: Inconsistent indentation:

```suggestion
        JSONRequest(domain + dsoPath + "wfs/?_format=json").catch(e => {console.log("Kan mvt datasets niet ophalen.")}),
        JSONRequest(domain + dsoPath + "mvt/?_format=json").catch(e => {console.log("Kan wfs datasets niet ophalen.")}),
        JSONRequest("/api/manual_apis.json").catch(e => {console.log("Kan manual datasets niet ophalen.")}),
```

File: static/dso_tables.js (hunk 1 above)
Comment (lbam) [reply static/dso_tables.js:6]:
```suggestion
 const dsoPath = "/v1/";
```

Comment (Yrob) [reply static/dso_tables.js:6]: woeps

**Review by lbam — APPROVED** [review]

**Pull request merged**